modin/engines/dask/pandas_on_dask/frame/partition.py

```diff
@@ -32,15 +50,21 @@ def apply_list_of_funcs(funcs, df):
 
 
 class PandasOnDaskFramePartition(BaseFramePartition):
-    """This abstract class holds the data and metadata for a single partition.
-    The methods required for implementing this abstract class are listed in
-    the section immediately following this.
-
-    The API exposed by the children of this object is used in
-    `BaseFrameManager`.
-
-    Note: These objects are treated as immutable by `BaseFrameManager`
-    subclasses. There is no logic for updating inplace.
```

Comment:
why these lines are gone?

Reply:
These went to rst (smth to BaseFramePartition, smth to PandasOnDaskFramePartition).